pom.xml

```diff
@@ -398,25 +407,40 @@
     </dependencyManagement>
 
     <dependencies>
-        <dependency>
-            <groupId>org.junit.jupiter</groupId>
-            <artifactId>junit-jupiter-api</artifactId>
-            <scope>test</scope>
-        </dependency>
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
             <version>4.13.2</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.junit.vintage</groupId>
+            <artifactId>junit-vintage-engine</artifactId>
+            <version>${jupiter.version}</version>
```

Comment:
This is not a `jupiter` dependency, so I think the version doesn't apply here.

Reply:
right, but there is no version for this dependency before, and 5.10.2 version also applied for this dependency. what i can do is to just use the number rather than a variable for this dependency?

Reply:
Yeah, that would be better, since you don't know if the versions will keep up.

Reply:
updated